scripts/run_db.sh

```diff
@@ -95,6 +95,14 @@ if [ "$SKIP_MIGRATE" = false ]; then
   migrate_db "$DSN"
 fi
 
-echo "DB is up!"
+echo "
+.__ .  .._.
+[__)|\/| | 
+|  \|  |_|_
+           
+.__ .__    
+|  \[__)   
+|__/[__)   
+"
```

Comment:
:grimacing:

Reply:
Do you have a pointer to the font you use for our other ones? I'd love for them to be consistent.

Reply:
We link to it! Is called Pagga:

https://github.com/RMI-PACTA/app/blob/2525f7f39e0748935a0f17b1174aa717a469c0da/cmd/server/main.go#L153

Reply:
<3